src/evm-block-extractor/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN cargo build --release --bin evm-block-extractor-server
 FROM debian:bookworm-slim AS runtime
 
 ENV DATASET=testnet
+ENV DATABASE=bigquery
```

Comment:
It should be `--bigquery`

Reply:
or change it in the code to be used without `--`

Reply:
If you check like 36; it is written like that.

Reply:
> I saw it, but line 36 is `--` `white_space` `bigquery` I don't know if ti works. Did you try it?

Yes; it does work